algorithms/kernel/dtrees/gbt/gbt_model_impl.h

```diff
@@ -185,6 +216,9 @@ class GbtDecisionTree: public SerializationIface
     size_t _sourceNumOfNodes;
     services::SharedPtr<SplitPointType> _splitPoints;
     services::SharedPtr<FeatureIndexesForSplitType> _featureIndexes;
+    services::Collection<size_t> nNodeSplitFeature;
```

Comment:
Usually  DynamicArray(TArray) is used for such purpose. But seems it`s fine.

Reply:
I think that the reason for usage Collection here - lack of CPU template in the class